src/multi_site_inputs_parser.py

```diff
@@ -130,6 +130,19 @@ def multi_site_csv_parser(path_to_csv, api_url, API_KEY, n_sites=None):
 
         path_to_load_files = "./load_profiles"
         add_load_profile_inputs(site_inputs, posts[-1], path_to_load_files=path_to_load_files)
+        
+        path_to_rate_files = "./electric_rates"
+        if "urdb_json_file" in df.columns:
+            if pd.isnull(df.loc[i,"urdb_json_file"]) == False:
+                rate_i = json.load(open(os.path.join(path_to_rate_files, df["urdb_json_file"][i]), "r"))
+                posts[i]["Scenario"]["Site"]["ElectricTariff"]["urdb_response"] = rate_i
+            elif pd.isnull(df.loc[i,"urdb_json_file"]) == True and pd.isnull(df.loc[i,"urdb_label"]) == True:
```

Comment:
What if a scenario has the ElectricTariff specified via inputs other than the `urdb_response` or `urdb_label`? (A user can provide [any of these sets](https://github.com/NREL/REopt_API/blob/fbc70f3b0cdeec9ee220266d6b3b0c5d64f257a6/reo/nested_inputs.py#L72)). I think that we can remove this `elif` block (otherwise we must account for every combination possible for specifying the `ElectricTariff`, which can change and thus will require maintenance). Is that okay with you @JesseKaczmarski ?

Reply:
Yes, if we remove the elif statement it will just revert to how the program operated before. I'll make the change!

Reply:
Thank you @JesseKaczmarski ! I will approve and merge.